src/types.db

```diff
@@ -129,6 +129,7 @@ latency                 value:GAUGE:0:U
 links                   value:GAUGE:0:U
 load                    shortterm:GAUGE:0:5000, midterm:GAUGE:0:5000, longterm:GAUGE:0:5000
 memory_bandwidth        value:DERIVE:0:U
+memory_maps             value:GAUGE:0:U
```

Comment:
Would the existing `file_handles` type also work? E.g. as "memory/file_handles-mapped"?

Reply:
Ah, I've added this to make it appear on top level in a descriptive name. How would I reference the file_handles type to achieve this?